pom.xml

```diff
@@ -48,7 +48,7 @@
         <dep.airlift.version>0.98</dep.airlift.version>
         <commons-httpclient.version>3.0.1</commons-httpclient.version>
         <dep.hadoop1.version>1.2.0</dep.hadoop1.version>
-        <dep.hadoop2.version>2.6.0</dep.hadoop2.version>
+        <dep.hadoop2.version>2.7.0</dep.hadoop2.version>
```

Comment:
Is the upgrade to 2.7.0 safe ? Why is this required ? Azure jars work with 2.7.0 only ?

Reply:
The NativeAzureFileSystem is available in hadoop 2.7. We are not referencing com.microsoft.azure for that.

Reply:
This needs to be checked for regression with currently supported versions of hive and presto

Reply:
I have changed the hadoop version back to 2.6 and added only 2.7 version dependency for hadoop-azure module. This is not available in 2.6